make judge-tier1 fails

Hi,
I was setting up judge using docker and followed the following steps:
$ git clone --recursive https://github.com/DMOJ/judge.git
```
Cloning into 'judge'...
remote: Enumerating objects: 20525, done.
remote: Counting objects: 100% (1492/1492), done.
remote: Compressing objects: 100% (584/584), done.
remote: Total 20525 (delta 999), reused 1229 (delta 885), pack-reused 19033
Receiving objects: 100% (20525/20525), 6.82 MiB | 5.33 MiB/s, done.
Resolving deltas: 100% (14347/14347), done.
```
$  cd judge-server/
$ make judge-tier1
The above command always fails when running setup.py develop for dmoj. The error message says "imporlib.metadata.PackageNotFoundError: dmoj". Here below is the detail message I received.

```
(dmojsite) ashis@ML-Lab:~/dmojca_installer$ cd judge/.docker
(dmojsite) ashis@ML-Lab:~/dmojca_installer/judge/.docker$ make judge-tier1
cd tier1 && docker build --build-arg TAG="master" -t dmoj/judge-tier1 -t dmoj/judge-tier1:latest .
Sending build context to Docker daemon  2.048kB
Step 1/4 : FROM dmoj/runtimes-tier1
 ---> cedfad3b8c9c
Step 2/4 : ARG TAG=master
 ---> Using cache
 ---> 97e622ccf49e
Step 3/4 : RUN mkdir /judge /problems && cd /judge && 	curl -L https://github.com/DMOJ/judge-server/archive/"${TAG}".tar.gz | tar -xz --strip-components=1 && 	pip3 install -e . && 	HOME=~judge . ~judge/.profile && 	runuser -u judge -w PATH -- dmoj-autoconf -V > /judge-runtime-paths.yml && 	echo '  crt_x86_in_lib32: true' >> /judge-runtime-paths.yml
 ---> Running in 6cbc3ded3610
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   125  100   125    0     0    273      0 --:--:-- --:--:-- --:--:--   272
100  805k    0  805k    0     0   528k      0 --:--:--  0:00:01 --:--:-- 4067k
Obtaining file:///judge
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
    Preparing wheel metadata: started
    Preparing wheel metadata: finished with status 'done'
Collecting termcolor
  Downloading termcolor-1.1.0.tar.gz (3.9 kB)
Collecting pylru
  Downloading pylru-1.2.0.tar.gz (18 kB)
Collecting watchdog
  Downloading watchdog-2.1.6-py3-none-manylinux2014_x86_64.whl (76 kB)
Collecting pygments
  Downloading Pygments-2.10.0-py3-none-any.whl (1.0 MB)
Collecting setproctitle
  Downloading setproctitle-1.2.2-cp39-cp39-manylinux1_x86_64.whl (35 kB)
Collecting pyyaml
  Downloading PyYAML-6.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (661 kB)
Building wheels for collected packages: pylru, termcolor
  Building wheel for pylru (setup.py): started
  Building wheel for pylru (setup.py): finished with status 'done'
  Created wheel for pylru: filename=pylru-1.2.0-py3-none-any.whl size=15732 sha256=1f29296eb048924b7a863c46bc990c27190b82ce110df2c71425226b0596a6c6
  Stored in directory: /root/.cache/pip/wheels/04/f4/30/a7c4d82b70acc60310d9ef4c1aeb9fb87b105a3cff1e418490
  Building wheel for termcolor (setup.py): started
  Building wheel for termcolor (setup.py): finished with status 'done'
  Created wheel for termcolor: filename=termcolor-1.1.0-py3-none-any.whl size=4847 sha256=be506cebece9f4765749730d3eb67600e566f3fc9dc3e0eb400badaeb6af9afe
  Stored in directory: /root/.cache/pip/wheels/b6/0d/90/0d1bbd99855f99cb2f6c2e5ff96f8023fad8ec367695f7d72d
Successfully built pylru termcolor
Installing collected packages: watchdog, termcolor, setproctitle, pyyaml, pylru, pygments, dmoj
  Running setup.py develop for dmoj
Successfully installed dmoj pygments-2.10.0 pylru-1.2.0 pyyaml-6.0 setproctitle-1.2.2 termcolor-1.1.0 watchdog-2.1.6
Traceback (most recent call last):
  File "/usr/bin/dmoj-autoconf", line 33, in <module>
    sys.exit(load_entry_point('dmoj', 'console_scripts', 'dmoj-autoconf')())
  File "/usr/bin/dmoj-autoconf", line 22, in importlib_load_entry_point
    for entry_point in distribution(dist_name).entry_points
  File "/usr/lib/python3.9/importlib/metadata.py", line 524, in distribution
    return Distribution.from_name(distribution_name)
  File "/usr/lib/python3.9/importlib/metadata.py", line 187, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: dmoj
The command '/bin/sh -c mkdir /judge /problems && cd /judge && 	curl -L https://github.com/DMOJ/judge-server/archive/"${TAG}".tar.gz | tar -xz --strip-components=1 && 	pip3 install -e . && 	HOME=~judge . ~judge/.profile && 	runuser -u judge -w PATH -- dmoj-autoconf -V > /judge-runtime-paths.yml && 	echo '  crt_x86_in_lib32: true' >> /judge-runtime-paths.yml' returned a non-zero code: 1
Makefile:9: recipe for target 'judge-tier1' failed
make: *** [judge-tier1] Error 1

```

Could you please help me figuring out the problem so that I can solve it? Thanks

- Ashis

--- a/.docker/tier1/Dockerfile
+++ b/.docker/tier1/Dockerfile
@@ -4,6 +4,7 @@ ARG TAG=master
 RUN mkdir /judge /problems && cd /judge && \
 	curl -L https://github.com/DMOJ/judge-server/archive/"${TAG}".tar.gz | tar -xz --strip-components=1 && \
 	pip3 install -e . && \
+	python3 setup.py develop && \
 	HOME=~judge . ~judge/.profile && \
 	runuser -u judge -w PATH -- dmoj-autoconf -V > /judge-runtime-paths.yml && \
 	echo '  crt_x86_in_lib32: true' >> /judge-runtime-paths.yml

--- a/.docker/tier2/Dockerfile
+++ b/.docker/tier2/Dockerfile
@@ -4,6 +4,7 @@ ARG TAG=master
 RUN mkdir /judge /problems && cd /judge && \
 	curl -L https://github.com/DMOJ/judge-server/archive/"${TAG}".tar.gz | tar -xz --strip-components=1 && \
 	pip3 install -e . && \
+	python3 setup.py develop && \
 	HOME=~judge . ~judge/.profile && \
 	runuser -u judge -w PATH -- dmoj-autoconf -V > /judge-runtime-paths.yml && \
 	echo '  crt_x86_in_lib32: true' >> /judge-runtime-paths.yml

--- a/.docker/tier3/Dockerfile
+++ b/.docker/tier3/Dockerfile
@@ -4,6 +4,7 @@ ARG TAG=master
 RUN mkdir /judge /problems && cd /judge && \
 	curl -L https://github.com/DMOJ/judge-server/archive/"${TAG}".tar.gz | tar -xz --strip-components=1 && \
 	pip3 install -e . && \
+	python3 setup.py develop && \
 	HOME=~judge . ~judge/.profile && \
 	runuser -u judge -w PATH -- dmoj-autoconf -V > /judge-runtime-paths.yml && \
 	echo '  crt_x86_in_lib32: true' >> /judge-runtime-paths.yml

